etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/addReceivingFacilityToMessageHeader.java

```diff
@@ -0,0 +1,33 @@
+package gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.custom;
+
+import gov.hhs.cdc.trustedintermediary.context.ApplicationContext;
+import gov.hhs.cdc.trustedintermediary.etor.metadata.EtorMetadataStep;
+import gov.hhs.cdc.trustedintermediary.etor.ruleengine.FhirResource;
+import gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.CustomFhirTransformation;
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiHelper;
+import gov.hhs.cdc.trustedintermediary.wrappers.MetricMetadata;
+import java.util.Collections;
+import java.util.Map;
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.MessageHeader;
+import org.hl7.fhir.r4.model.Organization;
+import org.hl7.fhir.r4.model.Reference;
+
+public class addReceivingFacilityToMessageHeader implements CustomFhirTransformation {
```

Comment:
I think this should be Receiving Application, not facility. Is the organization name the value we want to update here? Receiving Application doesn't have a name in HL7, it has namespace id, a universal id and universal id type. All those are in the organization identifier. Do we know which one do we need to replace? Do we need to collapse them into one?

Reply:
Looking at the spreadsheet, it seems we're collapsing them into MSH.5-1, which is the namespace (`identifier.where(extension.url = 'https://reportstream.cdc.gov/fhir/StructureDefinition/hl7v2Field' and extension.value = 'HD.1').value`)

Reply:
Pushed a rename calling it `ReceivingApplication` instead across the board.